scalafix-core/shared/src/main/scala/scalafix/internal/rule/Disable.scala

```diff
@@ -46,6 +46,36 @@ object Disable {
       buf.toList
     }
   }
+
+  def matchDisabledSymbol(disabledSymbol: DisabledSymbol, symbol: Symbol)(
```

Comment:
Would it be better to move this method to `DisabledSymbol.matches(Symbol)(Index)`?

I'm not a fan of pattern matching case classes with so many fields.